### DISCUSSION TITLE
Set a new AWS_SECRET_ACCESS_KEY ?

### DISCUSSION THREAD

**Discussion by olivier-sabban** [getlift/lift#293] [General]
How can I regenerate and set a new AWS_SECRET_ACCESS_KEY for the S3 bucket? 

The variable is not appearing in Lambda's environment variables. 

Thank you.

**Comment by olivier-sabban** [comment]
up ?

**Comment by ktwbc** [comment]
if the iam user exists, simplest way is just to go to the console and generate a new key/secret there (and revoke your old one). tbh you probably should be using something like secrets manager and reading the values from code, to avoid exposing them since env variables are easily viewable for anyone with simple lambda access.
